src/transactionNext.spec.ts

```diff
@@ -0,0 +1,223 @@
+import { assert } from "chai";
+import { TestWallet, loadTestWallets } from "./testutils";
+import { TransactionNext, TransactionComputer, Transaction } from "./transaction";
+import BigNumber from "bignumber.js";
+import { ProtoSerializer } from "./proto";
+
+class NetworkConfig {
+    MinGasLimit: number;
+    GasPerDataByte: number;
+    GasPriceModifier: number;
+    ChainID: string;
+
```

Comment:
Now I see that at some point we have to have consistent naming for public fields. Here, uppercase. For `TransactionNext`, the first letter is lowercase.

All right for now, no need to rename the fields of `NetworkConfig` in v13.

Reply:
Not really sure what you mean. The `TransactionComputer` uses the same `INetworkConfig` interface as the `Transaction` class where the first letter is uppercase.

Reply:
But I do agree that we need to stay consistent with the naming.